sys/include/net/ng_pktdump.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
  * @brief   Priority of the pktdump thread
  */
 #ifndef NG_PKTDUMP_PRIO
-#define NG_PKTDUMP_PRIO                 (PRIORITY_MIN - 1)
+#define NG_PKTDUMP_PRIO                 (PRIORITY_MAIN - 1)
```

Comment:
This was on purpose: https://github.com/RIOT-OS/RIOT/pull/2719#discussion_r27187941